controllers/operatorpolicy_controller.go

```diff
@@ -300,16 +333,16 @@ func buildOperatorGroup(
 ) *operatorv1.OperatorGroup {
 	operatorGroup := new(operatorv1.OperatorGroup)
 
-	gvk := schema.GroupVersionKind{
-		Group:   "operators.coreos.com",
-		Version: "v1",
-		Kind:    "OperatorGroup",
+	if policy.Spec.OperatorGroup == nil {
+		operatorGroup.SetGroupVersionKind(operatorgroupgvk)
+		operatorGroup.ObjectMeta.SetName(policy.Name + "-default-og")
+		operatorGroup.ObjectMeta.SetNamespace(policy.Spec.Subscription.Namespace)
+	} else {
+		operatorGroup.SetGroupVersionKind(operatorgroupgvk)
+		operatorGroup.ObjectMeta.SetName(policy.Spec.OperatorGroup.Name)
+		operatorGroup.ObjectMeta.SetNamespace(policy.Spec.OperatorGroup.Namespace)
+		operatorGroup.Spec.TargetNamespaces = policy.Spec.OperatorGroup.Target.Namespace
 	}
 
-	operatorGroup.SetGroupVersionKind(gvk)
-	operatorGroup.ObjectMeta.SetName(policy.Spec.Subscription.Package + "-operator-group")
-	operatorGroup.ObjectMeta.SetNamespace(policy.Spec.Subscription.Namespace)
-	operatorGroup.Spec.TargetNamespaces = []string{"*"}
-
 	return operatorGroup
```

Comment:
@JustinKuli , do you think it'd be worthwhile to add a label indicating ownership by the policy? I'm thinking it might help future reconciles to find the operator group. The same for the subscription.

Reply:
I think I'd prefer recording this information in the status of the OperatorPolicy. We'd already basically want that for RelatedObjects, I think.

In `enforce` mode, it would certainly be ok to add a label like that. But we'd have to consider if that kind of "write" operation would be ok in `inform` mode - and I think there would need to be similar considerations for `mustnothave` mode. If in any of those situations we *don't* want to add the label, then each situation might need to be handled differently... whereas if we always just record it in the status, and use it from the status, it's all somewhat consistent.

Reply:
That's fair. Since we will have a finalizer preventing the OperatorPolicy from being deleted and thus presumably, the status from being lost, I'm good with this.